templates/github-cd.yml

```
@@ -0,0 +1,35 @@
+      - name: Build Angular App
+        run: npm run ng build --configuration production --base-href /{{projectName}}/
+
+      - name: Angular Deploy gh-pages Actions
+        uses: JamesIves/github-pages-deploy-action@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: gh-pages
+          folder: dist/{{projectName}}
+
+permissions:
+  contents: write
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)